internal/scaffold/ansible/molecule_cluster_destroy.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ansible
+
+import (
+	"path/filepath"
+
+	"github.com/operator-framework/operator-sdk/internal/scaffold"
+	"github.com/operator-framework/operator-sdk/internal/scaffold/input"
+)
+
+const MoleculeClusterDestroyFile = "destroy.yml"
+
+type MoleculeClusterDestroy struct {
+	input.Input
+	Resource scaffold.Resource
+}
+
+// GetInput - gets the input
+func (m *MoleculeClusterDestroy) GetInput() (input.Input, error) {
+	if m.Path == "" {
+		m.Path = filepath.Join(MoleculeClusterDir, MoleculeClusterDestroyFile)
+	}
+	m.TemplateBody = moleculeClusterDestroyAnsibleTmpl
+	m.Delims = AnsibleDelims
+
+	return m.Input, nil
+}
+
+const moleculeClusterDestroyAnsibleTmpl = `---
+- name: Destroy
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  no_log: "{{ molecule_no_log }}"
+  tasks:
```

Comment:
Can you add a newline here (just to make it easier to see where the `tasks` begin)?